packages/orchestration/src/cosmos-api.ts

```diff
@@ -151,7 +151,9 @@ export interface StakingAccountActions {
    * The unbonding time is padded by 10 minutes to account for clock skew.
    * @param delegations - the delegation to undelegate
    */
-  undelegate: (delegations: Delegation[]) => Promise<void>;
+  undelegate: (
+    delegations: Omit<Delegation, 'delegatorAddress'>[],
```

Comment:
before release we should name these different types and define them additively (instead of subtracting)

but I think this is appropriate for now before we we've nailed down all the cases